packages/sign-plugin/tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
   "compilerOptions": {
+    "lib": ["es2023", "dom"],
+    "module": "node16",
+    "target": "es2022",
+    "moduleResolution": "node16",
```

Comment:
Until setting `"type": "module"` in sign-plugin package.json these changes will continue to output CJS. They are necessary to prevent failures in typechecking.